cli/cmd/integration.go

```diff
@@ -103,6 +106,16 @@ var (
 				return errors.New(msg)
 			}
 
+			integrationType, _ := api.FindIntegrationType(integration.Data[0].Type)
+			err = cli.LwApi.V2.Schemas.GetService(integrationType.Schema()).Get(args[0], &resp)
+
+			if err != nil {
+				cli.Log.Debugw("unable to get integration service")
```

Comment:
at least, print out the error message
```suggestion
				cli.Log.Debugw("unable to get integration service", "error", err.Error())
```